### PR TITLE
Basic profile page functionality

### PR DESCRIPTION
**Pull request by gmethvin**
It doesn't fully match the design yet but the functionality of editing name and description is there, and viewing social networks.

### CODE REVIEW

File: kifi-backend/modules/common/public/site/css/main.css [hunk 1]
```diff
@@ -99,6 +100,35 @@ a.query-x[href]:active {color: #556179}
...
+span.edit:before, span.save:before { display: inline-block; position: absolute; top: 0; left: -8px; content: "."; font-size: 0; height: 0; width: 0; border-top: 10px solid transparent; border-bottom: 10px solid transparent; border-right: 8px solid #b8c5d2; }
+.editing input {background: #e2edfc;}
+.edit-profile h2 {clear: both; border-bottom: 1px solid #818692; padding: 15px 0 0;}
+.edit-profile .networks li { display: inline-block; margin: 0 5px; width: 40px; height: 40px; font-size: 0; }
+.edit-profile .networks a { display: inline-block; margin: 0; padding: 0; width: 100%; height: 100%; }
+.edit-profile .nw-facebook { background: #637ab8 url(../img/networks/facebook.png) no-repeat scroll center center; background-size: 11px 21px; }
+.edit-profile .nw-linkedin { background: #4f83c3 url(../img/networks/linkedin.png) no-repeat scroll center center; background-size: 21px 20px; }
+.edit-profile .nw-twitter { background: #4f83c3 url(../img/networks/twitter.png) no-repeat scroll center center; background-size: 21px 17px; }
+.edit-profile .nw-google { background: #4f83c3 url(../img/networks/google.png) no-repeat scroll center center; background-size: 21px 21px; }
```
Comment (2is10) [inline kifi-backend/modules/common/public/site/css/main.css:129]: we generally want to sprite our images and produce retina versions. let's do it together once.

File: kifi-backend/modules/common/public/site/js/main.js [hunk 2]
```diff
@@ -1072,12 +1135,18 @@ $(function() {
...
+	}
+
 	// load data for persistent (view-independent) page UI
 	var promise = {
-		me: $.getJSON(urlMe, function(data) {
-			me = data;
-			$(".my-pic").css("background-image", "url(" + formatPicUrl(data.id, data.pictureName, 200) + ")");
-			$(".my-name").text(data.firstName + ' ' + data.lastName);
+		me: $.getJSON(urlMe, updateMe).promise(),
+		me: $.getJSON(urlNetworks, function (data) {
```
Comment (2is10) [inline kifi-backend/modules/common/public/site/js/main.js]: This networks promise should have a different name (e.g. `promise.myNetworks`). Or, if it isn't used anywhere, then we don't need to keep a reference to the promise.

File: kifi-backend/modules/common/public/site/js/main.js [hunk 3]
```diff
@@ -649,6 +707,11 @@ $(function() {
 		}
 		updateDetails();
 	});
+	$(".my-identity a").click(function (e) {
+		e.stopPropagation();
```
Comment (2is10) [inline kifi-backend/modules/common/public/site/js/main.js]: why is this needed? if not, let's remove it.

File: kifi-backend/modules/common/public/site/js/main.js (hunk 2 above)
Comment (gmethvin) [inline kifi-backend/modules/common/public/site/js/main.js]: oops. copy-paste mistake. I guess we don't need it everywhere so I can put it outside.

File: kifi-backend/modules/common/public/site/js/main.js (hunk 3 above)
Comment (gmethvin) [inline kifi-backend/modules/common/public/site/js/main.js]: Yeah I don't think we need it.

File: kifi-backend/modules/common/public/site/js/main.js [hunk 4]
```diff
@@ -177,6 +179,7 @@ $(function() {
 	});
 
 	var me;
+	var myNetworks;
```
Comment (2is10) [inline kifi-backend/modules/common/public/site/js/main.js:182]: `myNetworks` for clarity? We also load friends' networks for the friend hovercard.

**Comment by 2is10** [comment]
Merged. Feel free to address comments as you continue iterating on this. Thanks.
